Package.swift

```diff
@@ -25,13 +25,23 @@ let package = Package(
         .library(name: "onnxruntime",
                  type: .static,
                  targets: ["OnnxRuntimeBindings"]),
+        .library(name: "onnxruntime_extensions",
+                 type: .static,
+                 targets: ["OnnxRuntimeExtensions"]),
     ],
     dependencies: [],
     targets: [
         .target(name: "OnnxRuntimeBindings",
                 dependencies: ["onnxruntime"],
                 path: "objectivec",
-                exclude: ["ReadMe.md", "format_objc.sh"],
+                exclude: ["ReadMe.md", "format_objc.sh", "test",
+                            "ort_checkpoint.mm",
+                            "ort_checkpoint_internal.h",
+                            "ort_training_session_internal.h",
+                            "ort_training_session.mm",
+                            "include/ort_checkpoint.h",
+                            "include/ort_training_session.h",
+                            "include/onnxruntime_training.h"],
                 cxxSettings: [
                     .define("SPM_BUILD"),
```

Comment:
TODO: Need to update the macro def name for ORT as well when the changes are made.